src/diracx/routers/auth.py

```diff
@@ -348,6 +354,7 @@ async def initiate_device_flow(
             status_code=status.HTTP_400_BAD_REQUEST, detail="Unrecognised client ID"
         )
 
+    # TODO: This is an information leak
```

Comment:
what is ? The exception we return ?

Reply:
Ignore me, I didn't think about it properly 😄